### PR TITLE
Fabric interop

### PR DESCRIPTION
**Pull request by pmick**
# What's in this PR?

This PR renames the video native component to have a different prefix. RCT -> RNV

# Why did it change?

We decided to change this after testing some of our 3rd party components with fabric interop enabled. The fabric renderer's interop layer has some special case logic for components that are prefixed with RCT. In the react native docs they mention that the RCT prefix is reserved for components created by meta and 3rd parties should use their own prefix.

# How do I test it?

Follow the [README](https://github.com/discord/react-native-video/pull/2/files#diff-9702e1c1cd5228c116effc4953fac01a8ca3135f1b76f959cc0bc0a83217da98) in examples/fabric to build & run the project on iOS and Android. You can test with both fabric enabled & disabled if you'd like but we did that testing as well.

### CODE REVIEW

File: Video.js [hunk 1]
```diff
@@ -279,7 +279,7 @@ export default class Video extends Component {
     const isAsset = !!(uri && uri.match(/^(assets-library|ipod-library|file|content|ms-appx|ms-appdata):/));
 
     let nativeResizeMode;
-    const RCTVideoInstance = this.getViewManagerConfig('RCTVideo');
+    const RCTVideoInstance = this.getViewManagerConfig('RNVVideo');
```
Comment (hgarfinkle) [inline Video.js]: If we're changing RCTVideo to RNVVideo, we should probably rename the const here too

**Comment by hgarfinkle** [comment]
How do I test this?

File: Video.js (hunk 1 above)
Comment (pmick) [reply Video.js]: Done!

**Review by joemun — APPROVED** [review]
thank you!

**Pull request merged**